### ISSUE TITLE
v2.3 pop_eegfiltnew (line 148) command line call backward compatibility error?

### ISSUE THREAD

**Issue by ATC73bravo** [widmann/firfilt#2]
When running pop_eegfiltnew from cmd line with nargin = 7, the following error occurs:

```
Undefined function or variable 'revfilt'.

Error in pop_eegfiltnew (line 148)
    if nargin < 5 || isempty(revfilt)	
```

It seems the logic here is checking for an undefined variable. Commenting out lines 148-164 does the trick.

**Comment by widmann** [comment]
Arno, you introduced this code in the latest commit 1484940. I can replicate the problem. These variables are indeed not assigned before (and apparently also not used later on). Could please have a look into this? What was the intention?
Thanks!

**Comment by iras** [comment]
Same here for 'revfilt' and also 'usefft' after manually removing isempty(revfilt) at line 148

```
Undefined function or variable 'usefft'.

Error in pop_eegfiltnew (line 153)
    elseif usefft == 1```

**Comment by widmann** [comment]
Fixed by commit f3175c8.
